sql/conseil.sql

```diff
@@ -782,3 +782,72 @@ ALTER TABLE ONLY tezos.operations
 -- PostgreSQL database dump complete
 --
 
+CREATE SCHEMA bitcoin;
+
+-- https://developer.bitcoin.org/reference/rpc/getblock.html
+CREATE TABLE bitcoin.blocks (
+  hash text NOT NULL PRIMARY KEY,
+  confirmations integer NOT NULL,
```

Comment:
The number of confirmations can change rapidly, especially for more recent blocks. We should remove this for now and, in the future, add a separate `confirmations` table which is constantly updated.